examples/getreadme/main.go

```diff
@@ -0,0 +1,32 @@
+// Copyright 2016 The go-github AUTHORS. All rights reserved.
```

Comment:
@willnorris working on this, but I'm not familiar with the convention of an example in test. From [reading up here](https://blog.golang.org/examples), it looks like I may need to change the package to `github_test` in order to import and run this line: `client := github.NewClient(nil)`. If I do that, calls to `setup`, `teardown` and `RepositoryContent` are undefined. 

What's the right move here? Outputs below: 

```
### After changing the package to github_test ###
# github.com/google/go-github/github_test
./repos_contents_test.go:14: undefined: setup
./repos_contents_test.go:15: undefined: teardown
./repos_contents_test.go:16: undefined: RepositoryContent
./repos_contents_test.go:28: undefined: setup
./repos_contents_test.go:29: undefined: teardown
./repos_contents_test.go:30: undefined: RepositoryContent
./repos_contents_test.go:38: undefined: setup
./repos_contents_test.go:39: undefined: teardown
./repos_contents_test.go:40: undefined: mux in mux.HandleFunc
./repos_contents_test.go:41: undefined: testMethod
./repos_contents_test.go:41: too many errors
FAIL	github.com/google/go-github/github [build failed]

#### After importing and naming the package back to github ####
# github.com/google/go-github/github
import cycle not allowed in test
package github.com/google/go-github/github (test)
	imports github.com/google/go-github/github

FAIL	github.com/google/go-github/github [setup failed]
```